### DISCUSSION TITLE
入力ができないキーについての問い合わせ

### DISCUSSION THREAD

**Discussion by Forseti4** [Yowkees/keyball#410] [Q&A] [answered]
頻繁にお問い合わせして申し訳ありません。写真のソケットが認識されないのですが、その位置に対応するダイオードの位置を教えていただけますか？ それはEキーです。
![20231103_192601](https://github.com/Yowkees/keyball/assets/146462035/60868bf3-060f-4c2a-8411-982906ff522d)

**Comment by Forseti4** [comment]
ダイオードの問題ではなく、キーソケットのはんだが少なかったようです。解決しました。お手数をおかけして申し訳ありませんでした。

**Comment by Yowkees** [comment, marked as answer]
ご連絡が遅くなり申し訳ありません。
ご自身で解決いただきありがとうございました！ぜひ末永くご愛用いただければ幸いです。
